SWEBackend/SWEBackend/ClientApp/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
Why is all the code from the React App copied and pasted here? It already exists.

Reply:
This "backend" app is the entire hosted application, the "client" folder will be deleted and you guys will work out of the ClientApp folder within this sln